cmd/snap-seccomp/main.go

```diff
@@ -711,7 +711,8 @@ func compile(content []byte, out string) error {
 	if err != nil {
 		return err
 	}
-	defer fout.Close()
```

Comment:
Hm, silly (and probably tea deprived) question: could we override `Close()` somehow to make these errors less likely?